addons/mail/static/src/models/channel.js

```diff
@@ -160,6 +160,18 @@ registerModel({
             }
             return insertAndReplace({ category: replace(discussSidebarCategory) });
         },
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeHasCallFeature() {
+            const typesOfChannelWithCallFeature = new Set([
+                'channel',
+                'chat',
+                'group',
+            ]);
+            return typesOfChannelWithCallFeature.has(this.channel_type);
```

Comment:
Similar to previous comment, `Set` usage is actually less performant because `new Set()` is costly